web/src/lib/components/shared-components/create-share-link-modal/create-shared-link-modal.svelte

```diff
@@ -248,7 +233,7 @@
 	<hr />
 
 	<section class="m-6">
-		{#if !isShowSharedLink}
+		{#if !sharedLink}
```

Comment:
Would a flattened chain like this make more sense?

```svelte
{#if sharedLink}
{:else if editingLink}
{:else}
{/if}
```

Reply:
Maybe. I think there should be separate components for create vs edit myself.